psp/exp_configs/uk_pv_site_comp.py

```diff
@@ -0,0 +1,158 @@
+import datetime as dt
```

Comment:
Could you add a desrciption at the top, here, im not sure what `comp` or `comp_gen` means?